drf_spectacular/views.py

```diff
@@ -80,11 +82,25 @@ class SpectacularSwaggerView(APIView):
 
     @extend_schema(exclude=True)
     def get(self, request, *args, **kwargs):
+        static_absolute_path = "//unpkg.com/swagger-ui-dist"
+        swagger_bundle_js_href = "{}@{}/swagger-ui-bundle.js".format(static_absolute_path, spectacular_settings.SWAGGER_UI_DIST_VERSION)
```

Comment:
you can handle this substition in the template (handing through ``SWAGGER_UI_DIST``) with e.g. ``{{swagger_ui_dist}}/swagger-ui-bundle.js``